flask_rebar/rebar.py

```diff
@@ -56,6 +57,7 @@
 
 # To catch redirection exceptions, app.errorhandler expects 301 in versions
 # below 0.11.0 but the exception itself in versions greater than 0.11.0.
+# NOTE: as of Flask 1.0.3, redirects are no longer bubbled up as exceptions
```

Comment:
Not related in any way to current changes, but figured it wouldn't hurt to add some clarity re: recent shenanigans